extensions/rvadapter/src/main/java/com/google/android/agera/rvadapter/RepositoryPresenterCompiler.java

```diff
@@ -163,6 +184,7 @@ public int getLayoutResId(@NonNull final Object data, final int index) {
     public void bind(@NonNull final Object data, final int index,
         @NonNull final RecyclerView.ViewHolder holder) {
       binder.bind(getItems(data).get(index), holder.itemView);
+      collectionBinder.bind(data, holder.itemView);
```

Comment:
I prefer the collection binder called before the item binder, for reasons :)

But the `getItems(data)` call should still be called before calling either binder. So, local variable time.